src/dotfile.c

```diff
@@ -50,6 +50,22 @@
 #endif
 
 /*
+ * Checks whether a value is a boolean ("yes" or "no")
+ * Return 1 if "yes", 0 if "no", or -1 if not a boolean
+ * @val: value to check
+*/
+int
+is_boolean_value(const char* val)
```

Comment:
This is rather confusing. You use the return value of this function to print "light colors" but the function name has nothing to do about it.